moped-editor/src/components/GridTable/FiltersChips.js

```diff
@@ -33,19 +45,56 @@ const FiltersChips = ({ filters, isOr, filtersConfig }) => {
       (fieldConfig) => fieldConfig.name === filter.field
     );
     const fieldOperatorConfig = filtersConfig.operators[filter.operator];
+    const isDateType = fieldFilterConfig.type === "date";
     return {
       filterLabel: fieldFilterConfig?.label,
       operatorLabel: fieldOperatorConfig?.label,
-      filterValue: filter.value,
+      filterValue: isDateType ? formatDateType(filter.value) : filter.value,
     };
   });
 
+  /**
+   * Triggered by Filter Chip delete click
+   * Removes specified filter from state and update the url params
+   * @param {number} filterIndex - The index of the filter to be deleted
+   */
+  const handleDeleteButtonClick = (filterIndex) => {
+    const filtersNewState = [...filters];
+    filtersNewState.splice(filterIndex, 1);
+    setFilters(filtersNewState);
+
+    const remainingFiltersCount = filtersNewState.length;
+
+    if (remainingFiltersCount === 1) {
+      /* Reset isOr to false (all/and) if there is only one filter left */
+      setIsOr(false);
```

Comment:
@chiaberry i noticed in staging that the search param updates to `isOr=false` when the last filter is removed through the dropdown. I don't think this changes the behavior of this feature though since it flips to `isOr=false` when a second filter is applied, but it does create a brief mismatch between the search params (`isOr=true`) and the graphql query logic (`and`).

does it seem like a small fix to keep those values in sync?

Reply:
looks great!!!!